hyperdrive-cli/client/config.go

```diff
@@ -104,13 +119,7 @@ func (c *HyperdriveClient) UpdatePrometheusConfiguration(config *GlobalConfig) e
 }
 
 // Load the Grafana config template, do a template variable substitution, and save it
-func (c *HyperdriveClient) UpdateGrafanaDatabaseConfiguration(config *GlobalConfig) error {
-	// Make sure the metrics path exists
```

Comment:
Does this path always exist?

Reply:
If you mean the metrics path, yeah it created that dir as part of the method but I rolled that and the Prometheus one up into a single function to dedup. Check the latest commit in here.